front/src/reducers/osrdconf/types.ts

```diff
@@ -170,7 +171,9 @@ export type PathStepMetadata =
 
 export type StdcmPathStep = {
   id: string;
-  location?: Extract<OperationalPointReference, { uic: number }> & {
+  location?: {
+    reference: Extract<OperationalPointIdentifier, { uic: number }>;
+    track_reference?: null | TrackReference;
```

Comment:
Since we care only about the `uic` type, we can probably leave the rest as it was
```suggestion
  location?: Extract<OperationalPointIdentifier, { uic: number }> & {
```